doc/man3/SSL_CONF_cmd.pod

```diff
@@ -161,9 +161,10 @@ where applicable (e.g. B<X25519>, B<ffdhe2048>) or an OpenSSL OID name
 (e.g. B<prime256v1>). Group names are case sensitive. The list should be
 in order of preference with the most preferred group first.
 
-Currently supported groups for B<TLSv1.3> are B<P-256>, B<P-384>, B<P-521>,
-B<X25519>, B<X448>, B<ffdhe2048>, B<ffdhe3072>, B<ffdhe4096>, B<ffdhe6144>,
-B<ffdhe8192>.
+Groups for B<TLSv1.3> in the default provider are B<P-256>, B<P-384>,
+B<P-521>, B<X25519>, B<X448>, B<ffdhe2048>, B<ffdhe3072>, B<ffdhe4096>,
+B<ffdhe6144>, B<ffdhe8192>. Additional providers may make available
```

Comment:
We also support brainpoolP256r1tls13, brainpoolP384r1tls13 and  brainpoolP512r1tls13.

Reply:
Added them in latest commit. OK to omit the non-"...tls13" variants, though?

Reply:
Yes